lib/authorize.js

```diff
@@ -0,0 +1,52 @@
+import { getLogger } from 'appium-logger';
+import { exec } from 'teen_process';
+import {xcode as xcode} from '../lib/future';
```

Comment:
No need to go back, since we already in `lib`. Also put spaces around import curly braces.